src/FabPage.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import ComponentPage from './ComponentPage.js';
+import HeaderBar from './HeaderBar.js';
+import {MDCRipple} from '@material/ripple';
+
+import './styles/FabPage.scss';
+
+class Fab extends Component {
+  componentWillUnmount() {
+    this.ripple.destroy();
+  }
+
+  render() {
+    const {mini, exit} = this.props;
+    const classes = `mdc-fab material-icons ${mini ? 'mdc-fab--mini' : ''} ${exit ? 'mdc-fab--exited' : ''}`;
+
+    return (
+      <button
+        className={classes}
+        aria-label='Favorite'
+        ref={fabEl => {
+          if(fabEl) { this.ripple = new MDCRipple(fabEl); }
+        }}
+      >
+        <span className='mdc-fab__icon'>
+          favorite_border
+        </span>
+      </button>
+    );
+  }
+}
+
+class FabDemos extends Component {
+  state = {
+    exited: false,
+  };
+
+  render() {
+    return (
+      <div>
+        <h3 className='mdc-typography--subheading2'>Standard Floating Action</h3>
+        <Fab />
+
+        <h3 className='mdc-typography--subheading2'>Mini Floating Action</h3>
+        <Fab mini />
+
+        <h3 className='mdc-typography--subheading2'>Exiting Floating Action</h3>
```

Comment:
drop the "exiting floating action"....thats a demo. Its not necessary in a catalog